code/src/Album.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import data from "./data.json";
+import { Artist } from "./Artists";
+
+export const Album = () => {
+  return (
+    <div className="music-card">
+      {data.albums.items.map(item => {
+        return (
+          <div className="Album">
```

Comment:
How come you are writing it as `Album` and not `album`? Can't see that you are starting with a big letter in the other classNames :)